test.js

```diff
@@ -64,6 +64,19 @@ test('requires secure file permissions on .env', (t) => {
     t.is(err.message, `File permissions are unsafe. Fix: chmod 600 '${filepath}'`);
 });
 
+test('requires secure file permissions on .env in windows land', (t) => {
+    const previousPlatform = process.platform;
+    process.platform = 'win32';
+
+    const err = t.throws(() => {
+        fixture('unsafe-perm-windows-env-777');
+    }, Error);
+    const filepath = path.join('fixture', 'unsafe-perm-env-640', '.env');
```

Comment:
The filepath here doesn't match the fixture that was actually used.

Reply:
Oups sorry, went a bit too quick on this one and the previous one as well.